noir-projects/scripts/generate_vk_json.js

```diff
@@ -162,7 +178,8 @@ async function generateVKData(
   outputFolder,
   artifactPath,
   artifactHash,
-  isMegaHonk
+  isMegaHonk,
+  isRecursive
```

Comment:
Need to check this but we're ignoring this flag inside bb right? We should leave it off of the interface in that case.